content/blogposts/2022/batch-changes-ospo.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 'How our OSPO uses Batch Changes to automatically add license files'
```

Comment:
I kept hearing OSPO and had no idea what it was. I imagine >90% of people who read this title will also have no clue what it is.

Added to my infamous [WDYMWDIM](https://docs.google.com/spreadsheets/d/1E71lKEMl6sFvey0BAVWyQih2an64bSutZfBJ6haXY9c/edit#gid=0) sheet.

Reply:
Alternative title:

> How we analyzed hundreds of repositories to ensure they had open source licenses

Reply:
If OSPO is too niche then this post will be missing a large audience.

> How we analyzed hundreds of repositories to ensure they had open source licenses

I love it.